pipeline-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -7,13 +7,14 @@ def branches = [:]
 //and jenkins will unite them into a single run of the job
 
 for (int i = 0; i < 4; i++) {
+  def _i = i //if we tried to use i below, it would equal 4 in each job execution.
```

Comment:
I'm fine with changing the `_i'` to index, but as for linking to documentation, I don't know what I would link to other than the stack overflow link in the PR description? Do you want me to put that link in the code as a comment? Or do you have a better idea?

I'm pretty new to groovy myself, so I'm not sure what else there is to do here.